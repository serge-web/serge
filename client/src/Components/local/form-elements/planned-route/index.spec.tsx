/* global it expect */

import React from 'react'
import renderer from 'react-test-renderer'

import PlannedRoute from './index'

const handleChange = (command: string): void => {
  window.alert('Firing command:' + command)
}

it('PlannedRoute renders correctly', () => {
  const tree = renderer
    .create(<PlannedRoute handleCommand={handleChange} name="plannedRouteStatus" isMobile={true} status="pending" />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
