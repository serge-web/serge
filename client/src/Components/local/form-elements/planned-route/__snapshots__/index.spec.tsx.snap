// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlannedRoute renders correctly 1`] = `
<div
  className="planned-route"
>
  <button
    className="MuiButtonBase-root MuiButton-root MuiButton-contained"
    disabled={false}
    onBlur={[Function]}
    onClick={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex={0}
    type="button"
  >
    <span
      className="MuiButton-label"
    >
      Accept
    </span>
  </button>
  <button
    className="MuiButtonBase-root MuiButton-root MuiButton-contained"
    disabled={false}
    onBlur={[Function]}
    onClick={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex={0}
    type="button"
  >
    <span
      className="MuiButton-label"
    >
      Reject
    </span>
  </button>
   
</div>
`;
